app/views/unattended/provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -38,7 +38,7 @@ description: |
 hostname: <%= @host.name %>
 fqdn: <%= @host %>
 manage_etc_hosts: true
-users: {}
+users: []
```

Comment:
LGTM, but I have one question: How will this work for older versions of cloud-init? This will break the workflow for older versions, is that correct?